content/docs/guides/pipelines/pipelines-metrics.md

```diff
@@ -0,0 +1,35 @@
++++
+title = "Pipeline metrics"
+description = "Log and visualize pipeline metrics."
+weight = 3
+toc = true
+
+[menu.docs]
+  parent = "pipelines"
+  weight = 6
++++
+This page is for component author to log metrics from the component. For details about how to build your own component, please see [Editing Build Your Own Component](/docs/guides/pipelines/build-component).
+ 
+## Overview of Metrics
+Kubeflow Pipeline supports scalar metrics logging. Component author can write a list of metrics to describe the performance of the model to a local file which will later be uploaded as run-time metrics by the pipeline agent. The uploaded metrics will be visualized in the experiment runs table in the pipeline UI.
```

Comment:
logs and metrics are slightly different things, so using "log" as verb is a bit confusing.
I would suggest using "export" like prometheus, here and other places in this PR.
https://prometheus.io/docs/instrumenting/exporters/

Reply:
sg. done.